### ISSUE TITLE
Vim: Caught deadly signal VTALRM

### ISSUE THREAD

**Issue by cdosborn** [junegunn/vim-plug#592]
When I do `:PlugInstall | PlugClean` I get the following error message:
```
Vim: Caught deadly signal VTALRM
Vim: Finished.
```

I found that this issue **is** present in **master**, but **not** present in [0.9.1](https://github.com/junegunn/vim-plug/releases/tag/0.9.1).

Steps to reproduce:
```
# Fetch master
curl -fLo ~/.vim/autoload/plug.vim \
        --create-dirs https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim

# Copy a test vimrc
pbcopy <<EOF
call plug#begin('~/.vim/plugged')
Plug 'junegunn/vim-easy-align'
Plug 'kana/vim-textobj-user'
call plug#end()
EOF

# Run vim with test vimrc
vim -u <( pbpaste ) -c 'PlugInstall | PlugClean'
```

I found that the error only presented itself, when I had two or more plugins. Also the order of running the install before the clean seemed to matter also.

```
VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Jun 14 2016 16:06:49)
Compiled by root@apple.com
Normal version without GUI.  Features included (+) or not (-):
-arabic +autocmd -balloon_eval -browse +builtin_terms +byte_offset +cindent -clientserver -clipboard +cmdline_compl +cmdline_hist +cmdline_info +comments -conceal +cryptv
+cscope +cursorbind +cursorshape +dialog_con +diff +digraphs -dnd -ebcdic -emacs_tags +eval +ex_extra +extra_search -farsi +file_in_path +find_in_path +float +folding
-footer +fork() -gettext -hangul_input +iconv +insert_expand +jumplist -keymap -langmap +libcall +linebreak +lispindent +listcmds +localmap -lua +menu +mksession
+modify_fname +mouse -mouseshape -mouse_dec -mouse_gpm -mouse_jsbterm -mouse_netterm -mouse_sysmouse +mouse_xterm +multi_byte +multi_lang -mzscheme +netbeans_intg
-osfiletype +path_extra -perl +persistent_undo +postscript +printer -profile +python/dyn -python3 +quickfix +reltime -rightleft +ruby/dyn +scrollbind +signs +smartindent
-sniff +startuptime +statusline -sun_workshop +syntax +tag_binary +tag_old_static -tag_any_white -tcl +terminfo +termresponse +textobjects +title -toolbar +user_commands
+vertsplit +virtualedit +visual +visualextra +viminfo +vreplace +wildignore +wildmenu +windows +writebackup -X11 -xfontset -xim -xsmp -xterm_clipboard -xterm_save
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
      user exrc file: "$HOME/.exrc"
  fall-back for $VIM: "/usr/share/vim"
Compilation: gcc -c -I. -D_FORTIFY_SOURCE=0 -Iproto -DHAVE_CONFIG_H -arch i386 -arch x86_64 -g -Os -pipe
Linking: gcc -arch i386 -arch x86_64 -o vim -lncurses
```

- Type:
    - [x] Bug
    - [ ] Enhancement
    - [ ] Feature Request
    - [ ] Question
- OS:
    - [ ] All/Other
    - [ ] Linux
    - [x] OS X
    - [ ] Windows
- Vim:
    - [x] Terminal Vim
    - [ ] GVim
    - [ ] Neovim

**Comment by junegunn** [comment]
Thanks for the report. I can reproduce the problem. I compared the code with 0.9.1 and realized that a fix we made for earlier versions of Vim 8 (#537) is causing this problem on system Vim of macOS. I don't know the exact reason why the new code doesn't work – fundamentally it's a bug of system Vim – but I'll just patch vim-plug to use the old code on Vim 7 to workaround the problem.

**Comment by cdosborn** [comment]
Thanks for the quick response! Really appreciate the work you do.
